app/views/shopping_lists/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div>
+
+<h3 class="text-center"><%= current_user.name %>'s Shopping List</h3>
+
+<p class="text-center">Total Number of food items to buy: <span><strong><%= @recipe_foods.length %></strong></span>
+
+  Total value of food items needed:
+ <span><strong>$<%= @recipe_foods.map {|ingredient| ingredient.food.price * ingredient.quantity }.sum %></strong></span>
+</p>
```

Comment:
- [ ] Total value generated in this list does not correspond to the prices in the table below. Kindly fix this.

Reply:
- [ ] This issue has been resolved :heavy_check_mark: